test/functional/commands/find/by-uiautomator-e2e-specs.js

```diff
@@ -11,111 +12,119 @@ describe('Find - uiautomator', function () {
   let driver;
   before(async () => {
     driver = await initDriver(APIDEMOS_CAPS);
-    await driver.implicitWait(20000);
+    await B.delay(20000);
```

Comment:
is it the same as the line above?

Reply:
`driver` is different. Before it was an instance of the actual UiAutomator2 driver object, now it's an instance of `wd`. I'm not sure why we needed the implicitWait in the first place, I just refactored it.

Reply:
but driver.implicitWait != delay
These are two completely different things. @jlipps correct me if I'm wrong

Reply:
yes, these are two different things. `driver.implicitWait` sets the implicit wait timeout for a session. `B.delay` is just a sleep in the code. we want to keep the implicit wait here. In admc/wd I think it's something like `driver.setImplicitWaitTimeout(2000)`